packages/ui/cypress/tests/watched-acccounts/watched-accounts.cy.ts

```diff
@@ -0,0 +1,62 @@
+import { addresses } from '../../fixtures/accounts'
+import { localHost } from '../../fixtures/landingData'
+import { landingPage } from '../../support/page-objects/landingPage'
+import { settingsPage } from '../../support/page-objects/settingsPage'
+
+describe('Watched Accounts', () => {
+  it('can add an account to the watch list', () => {
+    cy.visit(localHost)
+    landingPage.watchAccountButton().click()
+    settingsPage.accountAddressInput().type(`${addresses.Alice}{enter}`)
+    // autocomplete on the input seems to affect reliability so we wait a bit
```

Comment:
This still failed for me :( like the address from Alice was truncated with the first `5`

Reply:
It seems to be the speed of cypress typing and when the autocomplete / dropdown appears that makes it wonky. I can slightly slow down the speed of typing and it helps. Best I can do at the moment as can't find any event, attribute or request to spy on.